app/components/avo/sidebar_profile_component.rb

```diff
@@ -1,11 +1,7 @@
 # frozen_string_literal: true
 
-class Avo::SidebarProfileComponent < ViewComponent::Base
-  attr_reader :user
-
-  def initialize(user:)
-    @user = user
-  end
+class Avo::SidebarProfileComponent < Avo::BaseComponent
+  prop :user, _Nilable(_Any), reader: :public
```

Comment:
I couldn’t figure out what to make this type. It seems to be called with `nil` in some of the tests. Otherwise it’s a User record, but it doesn't seem to be specifically an "Avo-User". So maybe it should be an `ActiveRecord::Base`? Can we be more specific? `Avo::BaseResource` or something, maybe?